packages/core/src/Site/index.ts

```diff
@@ -156,7 +157,7 @@ export class Site {
   toRebuild: Set<string>;
   externalManager!: ExternalManager;
   // TODO: add LayoutManager when it has been migrated
```

Comment:
Can delete this comment now :)

Reply:
@yucheng11122017  Thank you for your prompt review and encouraging message. The line deleted in the latest commit.